### PR TITLE
FIX: small fixes in emergency procedure

### LINKED ISSUE #144: Admonition not rendering in emergency procedure
Right above [manual extraction procedure](https://www.axonlab.org/hcph-sops/data-collection/emergency-procedures/#manual-extraction-procedure), and the code is below:

https://github.com/TheAxonLab/hcph-sops/blob/af28fa2b2962d4ddacdc625ce395314b59505757/docs/data-collection/emergency-procedures.md?plain=1#L89-L90

Could it be because the admonition is inside the bullet list ?

### PR DESCRIPTION
**Pull request by esavary**
- Correct the emergency procedure in case of fire (No need to roll the bed out of the room).
- Fix admonition bug. closes #144

### CODE REVIEW

**Review by acionca — APPROVED** [review]
Admonition seems fixed (checked locally)

File: docs/data-collection/emergency-procedures.md [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
     !!! danger "Cutting electricity will NOT turn off the magnetic field!"
 
-- [ ] Remove the subject from the scanner's room, applying the [manual extraction procedure described below](#manual-extraction-procedure).
+- [ ] Remove the subject from the scanner's room.
+If the bed is stuck inside the bore apply the first step of [manual extraction procedure described below](#manual-extraction-procedure) and help the participant leave the room.
```
Comment (oesteban) [inline docs/data-collection/emergency-procedures.md:10]: This change requires vetting from @celprov and @helenelajous. I'm not even sure I totally understand.

That said, I guess you meant to indent the second sentence:

```suggestion
- [ ] Remove the subject from the scanner's room.
    If the bed is stuck inside the bore, apply the first step of [manual extraction procedure described below](#manual-extraction-procedure) and help the participant leave the room.
```

The problem for me is what *If the bed is stuck inside the bore* actually means. What is in the mindset of anticipating such possibility.

**Review by oesteban — COMMENTED** [review]
I would prefer to have one PR fixing the missing newline. The other change is actually modifying an emergency measure and requires some discussion.

Can you create a new PR to only fix the admonition?

File: docs/data-collection/emergency-procedures.md (hunk 1 above)
Comment (esavary) [reply docs/data-collection/emergency-procedures.md:10]: I posted this in an issue #148

**Pull request closed without merging**